Formula/opencascade.rb

```diff
@@ -7,39 +7,43 @@ class Opencascade < Formula
   head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
 
   depends_on "cmake" => :build
-  depends_on "freetype"
+  depends_on "doxygen" => :build
   depends_on "freeimage"
-  depends_on "tbb"
+  depends_on "freetype"
   depends_on "gl2ps"
-  depends_on "tcl-tk"
-  depends_on "doxygen"
+  depends_on "tbb"
 
   def install
-    args = std_cmake_args
-    args << "-DUSE_GL2PS:BOOL=ON"
-    args << "-DUSE_FREEIMAGE:BOOL=ON"
-    args << "-DUSE_TBB:BOOL=ON"
-    args << "-DINSTALL_DOC_Overview:BOOL=ON"
-
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
     mkdir "build" do
-      system "cmake", "..", *args
+      system "cmake", "..", "-DUSE_GL2PS=ON",
+                            "-DUSE_FREEIMAGE=ON",
+                            "-DUSE_TBB=ON",
+                            "-DINSTALL_DOC_Overview=ON",
+                            "-D3RDPARTY_FREETYPE_DIR=#Formula['freetype'].opt_prefix",
+                            "-D3RDPARTY_FREEIMAGE_DIR=#Formula['freeimage'].opt_prefix",
+                            "-D3RDPARTY_FREEGL2PS_DIR=#Formula['gl2ps'].opt_prefix",
+                            "-D3RDPARTY_TBB_DIR=#Formula['tbb'].opt_prefix",
+                            "-D3RDPARTY_TCL_DIR:PATH=/usr",
+                            "-D3RDPARTY_TCL_INCLUDE_DIR:PATH=#{sdk}/usr/include",
+                            "-D3RDPARTY_TK_INCLUDE_DIR:PATH=#{sdk}/usr/include",
+                            *std_cmake_args
       system "make", "install"
     end
 
+    # Wrapper
+    (bin/"opencascade").write <<~EOS
+      #!/bin/sh
+      CASROOT=#{opt_prefix} exec #{opt_prefix}/bin/"$@"
+    EOS
+
     # Some apps expect resources in legacy ${CASROOT}/src directory
     cd prefix do
       ln_s "share/opencascade/resources", "src"
     end
   end
 
-  def caveats; <<~EOS
-    Some apps will require this enviroment variable:
-      CASROOT=#{opt_prefix}
-    EOS
-  end
-
   test do
-    ENV["CASROOT"] = opt_prefix
-    assert_match "1", shell_output("#{bin}/DRAWEXE -c \"pload ALL\"")
+    assert_equal "1", shell_output("#{bin}/opencascade DRAWEXE -c \"pload ALL\"").chomp
```

Comment:
```
    output = shell_output("#{bin}/opencascade DRAWEXE -c \"pload ALL\"")
    assert_equal "1", output.chomp
```

Reply:
I thought fewer lines == betterer? ;)  Fixed.

Reply:
not when they go passed the eightieth column :)